roles/spark/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Install Java package
```

Comment:
more roles than this are going to need Java. Could you create a separate Java role with this task and included it as a dependency in Spark?